node_modules/oae-preview-processor/lib/processors/file/pdf.js

```diff
@@ -308,3 +313,32 @@ var _convertToPNG = function(ctx, input, output, width, callback) {
         return callback();
     });
 };
+
+/**
+ * Convert a PDF file to plain text.
+ *
+ * @param  {PreviewContext}      ctx             The preview context associated to this file
+ * @param  {String}              input           The PDF file to convert
+ * @param  {String}              output          The path where the txt file should be written
+ * @param  {Function}            callback        Standard callback function
+ * @param  {Object}              callback.err    An error that occurred, if any
+ * @api private
+ */
+var _convertToText = function(ctx, input, output, callback) {
+    var cmd = util.format('pdftotext -nopgbrk -q "%s" "%s"', input, output);
```

Comment:
It'd be nice to have the following files:
 - plain.txt  - contains the plain text data for the entire pdf and can be used for search indexing
 - page.1.txt - Can be used by the UI as an accessibility hint for the first page
 - page.2.txt - ...
 - ..
 - page.n.txt